### PR TITLE
Update Font Awesome to v6.4.2

### LINKED ISSUE #365: Update the Discord logo to its latest version
### Description

The current Discord logo is the old Discord logo, which has been updated a few years back.
I believe the logo should be updated to the latest version.

If wanted, I'd like to work on this. 🙂

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Update Font Awesome to v6.4.2.
  - Closes #365.

### CODE REVIEW

File: forum/style/template/overall_footer.html [hunk 1]
```diff
@@ -127,31 +127,31 @@
 
 		<script type="text/javascript">
 			$( document ).ready(function($){
-				<!-- IF S_MERGE_VIEW -->
```
Comment (DomClark) [inline forum/style/template/overall_footer.html:130]: These aren't normal HTML comments, but rather phpBB template tags. Converting them to JavaScript comments will break the template. You'll need to revert all changes to this file.

**Review by DomClark — CHANGES_REQUESTED** [review]

**Review by DomClark — APPROVED** [review]

**Comment by Panquesito7** [comment]
Bump.

**Comment by tresf** [comment]
@Panquesito7 sorry for the delay on this.  Did you test this locally?  I just don't want to merge and add any major regressions (or at least have someone on the switch if some arise).  I'm just not sure how backwards compatible the icon names are.

**Comment by Panquesito7** [comment]
> @Panquesito7 sorry for the delay on this. Did you test this locally? I just don't want to merge and add any major regressions (or at least have someone on the switch if some arise). I'm just not sure how backwards compatible the icon names are.

Sure, no worries. Just tested it right now using [Gitpod](https://www.gitpod.io) and everything seems to be perfectly fine (at least on the main page). If you want to test it more easily, you can use Gitpod as well and run the necessary commands to compile and run the website online.

Updated (this PR).
image

Old (current).
image

**Pull request merged**